### ISSUE TITLE
gulp-ng-templates is not compatible with gulp-header 1.8.3

### ISSUE THREAD

**Issue by Huzzle** [VictorQueiroz/gulp-ng-templates#3]
If gulp-ng-templates is used like in [Example 1](https://github.com/VictorQueiroz/gulp-ng-templates#example-1) then nodejs throws error:

> Error: ENOENT: no such file or directory, lstat '/path/to/templates.js'

It is caused with recent changes in [gulp-header](https://github.com/tracker1/gulp-header) (v1.8.3). Please, adapt gulp-ng-templates to the latest version of [gulp-header](https://github.com/tracker1/gulp-header) or just fix 1.8.2 version in package.json

**Comment by ecozoic** [comment]
Yep we are having same issue. Should really be using strict versioning for all package.json dependencies. 

EDIT - As a workaround, we added a npm-shrinkwrap.json:

``` json
{
  "dependencies": {
    "gulp-ng-templates": {
      "version": "0.0.6",
      "dependencies": {
        "gulp-header": {
          "version": "1.8.2"
        }
      }
    }
  }
}
```

**Comment by jkneb** [comment]
Simply adding 

```
…
"gulp-header": "1.8.2",
…
```

to our main project package.json fixed the problem.

**Comment by VictorQueiroz** [comment]
Should be fixed on 0.0.9. Thank you guys for the waiting! 👍
